src/github.com/matrix-org/dendrite/common/test/strings.go

```diff
@@ -0,0 +1,32 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import "sort"
+
+// StringSliceEqual returns true if the slices have same length & elements.
```

Comment:
Probably worth noting that it does mutate the inputs

Reply:
yes, good catch.

Reply:
I should probably sort a copy of the slices.
since in tests we usually have small slices, this is okay.